src/components/header.tsx

```diff
@@ -1,109 +1,237 @@
+  };
+
+  const drawer = (
+    <div>
+      <Toolbar style={{ background: "#f89b27" }} />
+      <Divider style={{ background: "#f89b27" }} />
+      <List>
+        <ListItem disablePadding>
+          <ListItemButton component="a" href="https://advisory.sg">
+            <ListItemText primary="Back To Main Site" />
+          </ListItemButton>
+        </ListItem>
+        <ListItem disablePadding>
+          <ListItemButton
+            component="a"
+            href="#aboutus"
+            onClick={handleDrawerToggle}
           >
-            here
-          </a>{" "}
-          to apply as a mentee. Applications close{" "}
-          <b>17 September 2021, 2359h (GMT +8).</b>
-        </p>
-        <p>
-          Click{" "}
-          <a
-            target="_blank"
-            rel="noopener noreferrer"
-            href="https://drive.google.com/file/d/1w7zlMuGY_7hVXeW0LaNKXWWjLRt3ic4G/view?usp=sharing"
+            <ListItemText primary="About Us" />
+          </ListItemButton>
+        </ListItem>
+        <ListItem disablePadding>
+          <ListItemButton
+            component="a"
+            href="#sponsors"
+            onClick={handleDrawerToggle}
           >
-            here
-          </a>{" "}
-          to view the application guide on CVs and resumes.
-        </p>
-        <p>
-          Click{" "}
-          <a
-            target="_blank"
-            rel="noopener noreferrer"
-            href="https://docs.google.com/document/d/1mGjAsgTP7KIheWVDm-TZ5Hxxf8WYPfWemiqcZ40XCwo/edit?usp=sharing"
+            <ListItemText primary="Sponsors" />
+          </ListItemButton>
+        </ListItem>
+        <ListItem disablePadding>
+          <ListItemButton
+            component="a"
+            href="#mentors"
+            onClick={handleDrawerToggle}
           >
-            here
-          </a>{" "}
-          to view a list of FAQs for applicants.
-        </p>
-        <p>
-          Click{" "}
-          <a
+            <ListItemText primary="Mentors" />
+          </ListItemButton>
+        </ListItem>
+        <ListItem disablePadding>
+          <ListItemButton
+            component="a"
+            href="https://advisorysg.typeform.com/to/HQ8nWq2r#source=mentorsite"
             target="_blank"
             rel="noopener noreferrer"
-            href="https://drive.google.com/file/d/1aA3l_WQ1ehGO8UBKuRFLqj4URpQbAVzN/view?usp=sharing"
           >
-            here
-          </a>{" "}
-          to view a sample application response.
-        </p>
-        <p>
-          Click{" "}
-          <a
-            target="_blank"
-            rel="noopener noreferrer"
-            href="https://drive.google.com/file/d/19lUr2BchbYFAPMjEbN_M2mf1pyw-n7pj/view?usp=sharing"
+            <ListItemText primary="Apply Now" />
+          </ListItemButton>
+        </ListItem>
+      </List>
+      <Divider />
+    </div>
+  );
+  const container =
+    window !== undefined ? () => window().document.body : undefined;
```

Comment:
Please place these two lines of code directly into your code below, since this is fairly specific code and unlikely to be reused.